### DISCUSSION TITLE
AWX connection fail

### DISCUSSION THREAD

**Discussion by revolonl** [ansibleguy76/ansibleforms#169] [Q&A] [answered]
Hello all,
i have an issue with my AWX connection, when i try to Test the connection, i have an "AxiosError: Request failed with status code 502.
I use login/password to connect.
Regards.

**Comment by ansibleguy76** [comment]
A 502 bad gateway error is an HTTP status code that indicates a communication error between two web servers
Do you have routing set correctly ?  Can ping AWX from within your container ?
- proxy settings ?
- duplicate ip's ?
- ...

**Reply by revolonl** [reply, marked as answer]
yes i think that it's related to our proxy, but don't known where to configure it. Maybe someone can help me on this ;-)

**Comment by ansibleguy77** [comment]
Hello all,
The sample form “AWX Query” on Ansibleforms website, generates error “Should not have additional properties environment_execution” when "save" button is clicked. (with latest version -  5.0.3 ).

**Comment by ansibleguy76** [comment]
it has to be executionEnvironment :)

thx, I'll update the docs.
